hs/WebParsing/ParsingHelp.hs

```diff
@@ -1,8 +1,8 @@
 {-# LANGUAGE OverloadedStrings #-}
 module WebParsing.ParsingHelp
    (CoursePart,
-    (-:),
     (~:),
+    (-:),
```

Comment:
I'm not quite sure why these operators are necessary. They seem to just obfuscate function application.